organizations/organizations.shacl.ttl

```diff
@@ -539,6 +539,23 @@
     sh:description "L'adresse de l'emplacement."@fr ;
 
     sh:message "occurs more than once or its object is no schema:PostalAddress"@en ;
+  ],
+  [
+    a sh:PropertyShape ;
+    sh:path org:siteOf ;
+    sh:maxCount 1 ;
+    sh:minCount 1 ;
+    sh:class org:Organization ;
+
+    sh:name "site of"@en ;
+    sh:name "site de"@fr ;
+    sh:name "vestiging van"@nl ;
+
+    sh:description "De contentpartner waarvan deze vestiging deel uitmaakt."@nl ;
```

Comment:
Hier zouden we contentpartner best vervangen door organisatie. Hetzelfde geldt voor de andere talen